core-runner/src/runner.rs

```diff
@@ -244,7 +244,7 @@ impl<SC: StorageCarrier, E: Environment<Ext>> Runner<SC, E> {
     ///
     /// Runner will return actual number of messages that was handled.
     /// Messages with no destination won't be handled.
-    pub fn run_next(&mut self, message: Message) -> RunNextResult {
+    pub fn run_next(&mut self, message: Message, next_entry: RunNextEntry) -> RunNextResult {
```

Comment:
Can we consider making the entry point a part of the Message itself? Since the runner needs to know what the entry point is anyway...

Reply:
Good point. I'd prefer to implement it within another task =).

What do you think @shamilsan @breathx @NikVolf ?

Reply:
This is already done in runner refactoring